### ISSUE TITLE
Documentation update

### ISSUE THREAD

**Issue by damms005** [vercel/hazel#135]
For folks like us that have never used Vercel before until we encounter this awesome Hazel project, the sentence in the README that reads `You only need to click a button` is really confusing because after clicking the supposed `Deploy` button and setting everything up, it is required to still click the dropdown that reads `PROMOTE TO PRODUCTION` on the Vercel dashboard for the project.

The `You only need to click a button` phrase is indeed correct and aptly describes the ease of deployment. However, I want to suggest that somewhere in the README, the `PROMOTE TO PRODUCTION` step should be mentioned too, to save debug time for Vercel-newbies like myself.

Awesome project, thanks a lot!

**Comment by ayorich** [comment]
yes, documentation wasn't clear on this. have to stumble on this before I knew there wasn't actually an error as it fails to run till I promote it to production. thanks
